biogeophys/FatesHydroWTFMod.F90

```diff
@@ -163,6 +165,36 @@ module FatesHydroWTFMod
      procedure :: dftcdpsi_from_psi => dftcdpsi_from_psi_cch
      procedure :: set_wkf_param     => set_wkf_param_cch
   end type wkf_type_cch
+  ! =====================================================================================
+  ! Type1 Smooth approximation of Clapp-Hornberger and Campbell (CCH) water retention and conductivity functions
+  ! Bisht et al. Geosci. Model Dev., 11, 4085–4102, 2018
+  ! =====================================================================================
+
+  ! Water Retention Function
+  type, public, extends(wrf_type) :: wrf_type_smooth_cch
+     real(r8) :: th_sat   ! Saturation volumetric water content         [m3/m3]
+     real(r8) :: psi_sat  ! Bubbling pressure (potential at saturation) [Mpa]
+     real(r8) :: beta     ! Clapp-Hornberger "beta" parameter           [-]
+     real(r8) :: scch_pu, scch_ps, scch_b2, scch_b3
```

Comment:
Please provide descriptions of these parameters

Reply:
I have added variable definition in [PR#766](https://github.com/NGEET/fates/pull/766). Should we ignore this PR and work on [PR#766](https://github.com/NGEET/fates/pull/766) only?

Reply:
if all of the content you want to submit is contained in #766, then yes it should be ok to close this PR.  I would hold on to the branches, don't delete them, in case we want to use them.
Although I haven't read through 766 in detail, my only concern is that we should not completely remove the 1D solve, at least not without a team discussion.  I'm not against making 2D the default solver.  Do you remove the 1D solver code in 766?

Reply:
Yes, I removed the 1D in 766.